.mega-linter.yml

```diff
@@ -24,5 +26,6 @@ SPELL_LYCHEE_FILTER_REGEX_EXCLUDE: (\.github|\.vscode)
 SHOW_ELAPSED_TIME: true
 
 FILEIO_REPORTER: false
+
 # TODO Enable this (but it will likely require some more work) - at least once, then disable again? (Because it won't be constantly needed, once it's clean.)
-# VALIDATE_ALL_CODEBASE: true
+VALIDATE_ALL_CODEBASE: true
```

Comment:
According to https://github.com/oxsecurity/megalinter#common-variables this is apparently already the default anyway...